config/default_records/plate_purposes/009_bioscan_purposes.yml

```diff
@@ -12,11 +12,18 @@ LILYS-96 Stock:
   type: PlatePurpose::Input
   stock_plate: true
   cherrypickable_target: false
-# This is the lysed material plate, ready for the Library prep pipeline. These plates
-# are either prepared by the faculty team manually, or are created on the Beckman robot
-# from the LILYS-96 Stock parent.
-# These plates include controls.
+# This is the lysed material plate, ready for the Library prep pipeline.
+# This version of the Lysate plate is created on the Beckman robot from
+# the LILYS-96 Stock parent.
+# These plates include 2 added controls.
 LBSN-96 Lysate:
-  type: PlatePurpose::AdditionalInput
+  stock_plate: false
+  cherrypickable_target: false
+# This is the lysed material plate, ready for the Library prep pipeline.
+# This version of the Lysate plate is created by the faculty team manually,
+# and input via manifest.
+LBSN-96 Lysate Input:
+  type: PlatePurpose::Input
+  input_plate: true
```

Comment:
Separate type attribute is redundant if input_plate is set. See app/models/plate_purpose.rb